### DISCUSSION TITLE
Reuse opened Chrome tab

### DISCUSSION THREAD

**Discussion by erikmartinessanches** [ttscoff/bunch-docs#312] [Q&A] [answered]
1. I can use `%chrome:someURL` so open a new chrome tab. However, if the tab is already open, is there some way to simply show that tab, rather than opening a new one?

2. Another question, relating to profiles: In Chrome the profiles seem to be "First Name Last Name (email)" rather than "personal", as [the docs](https://bunchapp.co/docs/bunch-files/opening-web-pages/#opening-in-a-specific-chrome-profile) show. I tried `%chrome[personal]:someURL` and `%chrome[email]:someURL` but that crashed Bunch. So not quite sure how to use Chrome profiles with Bunch. This isn't as important to me as I tend to only use one profile.

**Comment by ttscoff** [comment, marked as answer]
No, Bunch has no knowledge of what tabs are open at the time, it will 
just send an open command. And profiles _should_ find the best matching 
profile for whatever string you give it. It's a tricky hack to make it 
work at all, though, so your mileage may vary.

-Brett

**Comment by erikmartinessanches** [comment]
I was able to achieve reusing open tabs with [this chrome extension](https://chromewebstore.google.com/detail/clutter-free-prevent-dupl/iipjdmnoigaobkamfhnojmglcdbnfaaf).

**Comment by ttscoff** [comment]
Nice. Now I just have to figure out what's going on with Profiles. I'll 
add this extension to the docs.

**Reply by ttscoff** [reply]
[Added to docs](https://bunchapp.co/docs/bunch-files/opening-web-pages/#preventing-duplicate-tabs)
